### DISCUSSION TITLE
winget install command

### DISCUSSION THREAD

**Discussion by PhilippBaumi** [microsoft/winget-cli#1521] [Ideas] [answered]
Why is the following command not available to install a program on another partition?

`winget install <tool> partition=<Partition>`

You don't need to specify anything for partition C because it is C by default, but the command would be useful for another. 

Can you also install UWP apps (i.e. those from the store) via winget?

**Comment by denelon** [comment, marked as answer]
https://docs.microsoft.com/windows/package-manager/winget/install

I believe you are looking for the `--location` option.

Yes, the Microsoft Store is now a source for the Windows Package Manager.

**Reply by PhilippBaumi** [reply]
How do you have to specify the location in the command?

**Reply by denelon** [reply]
As long as the installer supports passing the location as an option, you would specify the path you want the package installed in. 

`winget install <package> --location <path>`

**Reply by PhilippBaumi** [reply]
Is the following command possible?

`winget import -i <import-file> —location <path>`

If not, how can I use the import command to control where it should be installed?

**Reply by denelon** [reply]
* https://github.com/microsoft/winget-cli/issues/1582
